src/server/warnings/index.js

```diff
@@ -0,0 +1,9 @@
+export default function (kbnServer, server, config) {
```

Comment:
I'm assuming that we only have one instance of the kbnServer per 'process' (including when we're clustering) so we won't be subscribing to this event twice and possibly getting duplicate warnings.